lua/lvim/plugins.lua

```diff
@@ -60,19 +60,23 @@ local core_plugins = {
     end,
     requires = {
       "L3MON4D3/LuaSnip",
-      "rafamadriz/friendly-snippets",
+      { "rafamadriz/friendly-snippets", opt = not lvim.builtin.luasnip.sources.friendly_snippets },
```

Comment:
```suggestion
```

I think this should've been `disable = ...`, but cmp doesn't require this anyway, so might as well remove it.